dist/common/scripts/scylla_setup

```diff
@@ -176,6 +181,20 @@ def current_umask():
     os.umask(current)
     return current
 
+def update_rsysconf(rsyslog_server):
+    if ':' not in rsyslog_server:
+        rsyslog_server = rsyslog_server + ':1514'
+
+    with open('/etc/rsyslog.conf', 'r') as f:
+        for l in f:
+            if l.strip().startswith('#'):
+                continue
+            if rsyslog_server in l:
+                return
```

Comment:
Can we use /etc/rsyslog.d instead of editing files?

Reply:
It's a good idea, I'll switch to that, it removes the corner cases